jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/CreateLambdaPanel.java

```diff
@@ -3,19 +3,22 @@
 
 package software.aws.toolkits.jetbrains.services.lambda.upload;
 
+import com.intellij.openapi.project.Project;
 import com.intellij.openapi.ui.ComboBox;
+import com.intellij.util.textCompletion.TextFieldWithCompletion;
 import javax.swing.JButton;
 import javax.swing.JPanel;
 import javax.swing.JTextField;
 import org.jetbrains.annotations.NotNull;
 import software.amazon.awssdk.services.lambda.model.Runtime;
+import software.aws.toolkits.jetbrains.services.lambda.HandlerCompletionProvider;
 import software.aws.toolkits.jetbrains.ui.EnvironmentVariablesTextField;
 
 @SuppressWarnings("NullableProblems")
 public final class CreateLambdaPanel {
     @NotNull JTextField name;
     @NotNull JTextField description;
-    @NotNull JTextField handler;
+    @NotNull com.intellij.ui.EditorTextField handler;
```

Comment:
Why is the EditorTextField fully qualified name? Can we just use the simple name?

Reply:
The GUI editor did the change, didn't notice